packages-exp/auth-exp/src/platform_browser/auth_window.d.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** Extend the Window type to include arbitrary keys (to avoid use of `any`) */
+export interface AuthWindow extends Window {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  [key: string]: any;
```

Comment:
do we want to whitelist the keys we expect?

Reply:
Done